modules/tinymce/src/core/main/ts/api/dom/ScriptLoader.ts

```diff
@@ -118,7 +118,10 @@ class ScriptLoader {
 
       const crossOrigin = this.settings.crossOrigin;
       if (Type.isFunction(crossOrigin)) {
-        dom.setAttrib(elm, 'crossorigin', crossOrigin(url));
+        const resultCrossOrigin = crossOrigin(url);
+        if (resultCrossOrigin !== undefined) {
+          dom.setAttrib(elm, 'crossorigin', resultCrossOrigin);
+        }
```

Comment:
You don't actually need this. `setAttrib` will remove the value when set to undefined, null, or empty string. This ticket is just updating the customer-facing type to match the implementation.

- [ ] revert